content/webapp/components/EventCard/EventCard.tsx

```diff
@@ -1,25 +1,28 @@
+import { FC } from 'react';
 import { EventBasic } from '../../types/events';
 import CompactCard from '../CompactCard/CompactCard';
 import Image from '@weco/common/views/components/Image/Image';
 import StatusIndicator from '@weco/common/views/components/StatusIndicator/StatusIndicator';
 import EventDateRange from '../EventDateRange/EventDateRange';
 import { classNames, font } from '@weco/common/utils/classnames';
 import { getCrop } from '@weco/common/model/image';
+import { fixEventDatesInJson } from '../../services/prismic/transformers/events';
 
 type Props = {
   event: EventBasic;
   xOfY: { x: number; y: number };
 };
 
-const EventCard = ({ event, xOfY }: Props) => {
-  const DateRangeComponent = <EventDateRange event={event} />;
+const EventCard: FC<Props> = ({ event, xOfY }) => {
+  const eventWithDates = fixEventDatesInJson(event);
```

Comment:
nit: I think something like this will create a `jsonEvent` variable:

```suggestion
const EventCard: FC<Props> = ({ event: jsonEvent, xOfY }) => {
  const event = fixEventDatesInJson(jsonEvent);
```

It's a small thing, but I slightly prefer it because it makes it obvious which variable you *shouldn't* use, and why.